kinetic/distribution.yaml

```diff
@@ -4256,6 +4256,13 @@ repositories:
       url: https://github.com/HebiRobotics/hebi_cpp_api_ros.git
       version: master
     status: developed
+  hebi_description:
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/HebiRobotics/hebi_description-release.git
+      version: 0.1.0-1      
+    status: developed
```

Comment:
Same problem as described in https://github.com/ros/rosdistro/pulls#discussion_r322711027